src/transformers/models/whisper/generation_whisper.py

```diff
@@ -482,6 +483,11 @@ def generate(
             force_unique_generate_call (`bool`, *optional*):
                 Whether to force a unique call to the underlying GenerationMixin's [`~generation.GenerationMixin.generate`] method. This is useful for assisted decoding and testing purposes to ensure
                 that only one call to [`~generation.GenerationMixin.generate`] is made and therefore decoder input token ids and eos token ids are returned.
+            monitor_progress (`Callable[[torch.Tensor], None]`, *optional*):
+                If provided, this function is called to report the progress of the audio transcription. The function
+                takes one tensor argument p of shape (n, 2), where n is the batch size. p[i, 0]  contains the
+                index of the audio frame that is currently being transcribed for batch item i. p[i, 1] contains
+                the total number of frames for batch item i. No return value is expected.
```

Comment:
Thanks for the docstring! Could you add `` around the variables:
- `p`
- `(n, 2)`
- `n`
- `i`
- `p[i, 0]`
- `p[i, 1]`

And small rewording:
- "this function _can be_ called to report..."
- "function tasks _a_ tensor argument `p` of shape..."

thanks!